### PR TITLE
Add checksum for py-google-auth 2.11.0, py-cachetools 5.2.0

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
Confirmed sha256.

**Comment by iarspider** [comment]
@spackbot run pipeline

**Comment by iarspider** [comment]
@tldahlgren should I swap when and type here as well?

**Comment by tldahlgren** [comment]
> @tldahlgren should I swap when and type here as well?

That would be great.

**Review by tldahlgren — DISMISSED** [review]
LGTM

**Comment by tldahlgren** [comment]
Deferring merge to give @adamjstewart a chance to comment.

File: var/spack/repos/builtin/packages/py-google-auth/package.py [hunk 1]
```diff
@@ -13,19 +13,21 @@ class PyGoogleAuth(PythonPackage):
...
     pypi = "google-auth/google-auth-1.6.3.tar.gz"
 
+    version("2.11.0", sha256="ed65ecf9f681832298e29328e1ef0a3676e3732b2e56f41532d45f70a22de0fb")
     version("2.3.2", sha256="2dc5218ee1192f9d67147cece18f47a929a9ef746cb69c50ab5ff5cfc983647b")
     version("1.6.3", sha256="0f7c6a64927d34c1a474da92cfc59e552a5d3b940d3266606c6a28b72888b9e4")
 
     depends_on("python@2.7:2.8,3.4:", type=("build", "run"))
-    depends_on("python@3.6:", type=("build", "run"), when="@1.35:")
-    depends_on("python@2.7:2.8,3.6:", type=("build", "run"), when="@1.24:")
+    depends_on("python@3.6:", when="@1.35:", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-google-auth/package.py]: Even in 2.11.0 it still says 2.7 is supported

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**